VVResonances/python/statistics/Fitter.py

```diff
@@ -466,7 +491,7 @@ def signalResonanceCBGaus(self,name = 'model',poi="MVV",mass=0):
 	gsigma = ROOT.RooFormulaVar("gsigma","gsigma","@0*@1", ROOT.RooArgList(self.w.var('SIGMA'),self.w.var('SCALESIGMA')))
 	getattr(self.w,'import')(gsigma,ROOT.RooFit.Rename('gsigma'))
         self.w.factory("ALPHA[0.85,0.60,1.20]")
-        self.w.factory("N[126.9]") #From @dani: N=126
+        self.w.factory("N[7]") #From @dani: N=126 from thea 126.9
```

Comment:
We need to make some changes here to be able to run all signals with same script